### PR TITLE
Housekeeping: Reconfigure Chromatic

### LINKED ISSUE #3388: [Testing] Re-configure Chromatic CI workflow
- [x] ~~Run chromatic on Github action push event to avoid losing baselines: https://www.chromatic.com/docs/github-actions/#recommended-configuration-for-build-events~~
  - Seems like this is unneccessary, I have enabled the Chromatic Github app instead, which should carry over the baselines: https://www.chromatic.com/docs/github-actions/#github-squashrebase-merge-and-the-main-branch
- [x] How do we avoid running snapshot tests for each and every push to a PR?
  - Only run before merge of every PR? A label on PR?
- [ ] Configure Github branch protection rules to require status checks to pass before merging - both `chromatic` and `UI Tests`
- [x] Enable [UI Review powered by stories](https://www.chromatic.com/docs/storybook/review/)

### PR DESCRIPTION
**Pull request by RasmusKjeldgaard**
## Which issue does this PR close?

This PR closes #3388

## What is the new behavior?
- [x] Workflow trigger is changed from `pull_request` to `push`, as per chromatic recommendations and to work with turbosnap at a later point: https://www.chromatic.com/docs/setup-turbosnap/#compatibility
- [x] Fonts are preloaded, as snapshots would sometimes be taken before font was loaded
- [x] Simplified storybook build targets in project.json so storybook server now uses build-storybook config
  - [x] Also added `"enableProdMode": false` to storybook config to work around this issue: https://github.com/storybookjs/storybook/issues/14828
- [ ] When this PR is merged: Set chromatic as required checks for develop branch

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Reminders
- [x] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [x] Make sure you have updated the cookbook with examples and showcases (for bug fixes, enhancements & new components).

### Review  
- [x] Determine if your changes are a fix, feature or breaking-change, and add the matching label to your PR. If it is tooling, dependency updates or similar, add ignore-for-release.
- [x] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review).
- [x] Request that the changes are code-reviewed 
- [ ] Request that the changes are [UX reviewed](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#ux-review) (only necessary if your PR introduces visual changes)

When the pull request has been approved it will be merged to `develop` by Team Kirby.

### CODE REVIEW

File: .github/workflows/chromatic.yml [hunk 1]
```diff
@@ -17,7 +17,40 @@ jobs:
...
+        id: check_storybook_folders
+        run: |
+          if [ -d ${{ github.workspace }}/dist/storybook/designsystem ]; then
+            echo "designsystem=true" >> $GITHUB_OUTPUT
+          fi
+          if [ -d ${{ github.workspace }}/dist/storybook/extensions/angular ]; then
+            echo "extensions_angular=true" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Publish Designsystem  to Chromatic
```
Comment (jkaltoft) [inline .github/workflows/chromatic.yml:38]: Not important: Two spaces between words

```suggestion
      - name: Publish Designsystem to Chromatic
```

File: .github/workflows/chromatic.yml [hunk 2]
```diff
@@ -17,7 +17,40 @@ jobs:
...
+      - name: Publish Extensions to Chromatic
+      # If extensions storybook dist has been built, it means project might have been affected and should be tested
+        if: steps.check_storybook_folders.outputs.extensions_angular == 'true'
         uses: chromaui/action@latest
         with:
-          projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
+          projectToken: ${{ secrets.EXTENSIONS_CHROMATIC_PROJECT_TOKEN }}
+          workingDir: libs/extensions/angular
+          storybookBuildDir: ${{ github.workspace }}/dist/storybook/extensions/angular
+          autoAcceptChanges: "develop"
```
Comment (jkaltoft) [inline .github/workflows/chromatic.yml:56]: Not important: Missing EOF line-break

**Review by jkaltoft — APPROVED** [review]
Great housekeeping!

giphub

I only have a few comments/questions:

- Should we remove the `chromatic` label from Github? Or do we use it for other purposes?
- For consideration: Will it be an obstacle that we can't push changes to a PR without running snapshot tests? (Probably a premature concern 😊)

Once again, nice work!

🎗️ Don't forget to configure Github branch protection rules

**Review by jakobe — APPROVED** [review]
Nice work!

**Pull request merged**